.github/workflows/integration.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: |
           cargo nextest run \
             --locked --features "asm-keccak ${{ matrix.network }}" \
-            --workspace --exclude examples --exclude ef-tests \
+            --workspace --exclude ef-tests \
```

Comment:
Do we check examples anywhere else? I would like to fix this instead of removing it to not compile examples for testing purposes